website/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "@docusaurus/core": "2.0.0-beta.1",
     "@docusaurus/preset-classic": "2.0.0-beta.1",
+    "@docusaurus/theme-search-algolia": "^2.0.0-beta.4",
```

Comment:
^ is regex?

Reply:
it means exact match, auto added by `npm install --save xxx`